src/main/resources/swagger/api-docs.yaml

```diff
@@ -42,14 +38,13 @@ tags:
 ## PATHS
 ##########################################################################################
 paths:
-
-  '/api/ping':
+  /api/ping:
     get:
       summary: Ping
       responses:
-        200:
+        '200':
           description: Pong
-        500:
+        '500':
```

Comment:
I was going to ask if the quotes are really required but the Swagger spec says yes: https://swagger.io/specification/#patterned-fields-69 Very thorough!

Reply:
Does this solve the "Swagger works 50% of the time" problem?

Reply:
swagger seems to work 100% of the time now and i'm still not sure why but 🤷‍♂️